dll/win32/rasdlg/lang/ro-RO.rc

```diff
@@ -213,14 +213,14 @@ CAPTION "Conectare %1"
 FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL 1678, 1100, "STATIC", SS_BITMAP, 0, 0, 263, 50
-    LTEXT "A&pelează:", 1412, 9, 60, 68, 8
+    LTEXT "A&pelare:", 1412, 9, 60, 68, 8
     COMBOBOX 1416, 88, 56, 165, 112, CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_VSCROLL | NOT WS_TABSTOP
-    LTEXT "Apelează &din:", 1415, 9, 77, 71, 8
+    LTEXT "Apelare &din:", 1415, 9, 77, 71, 8
     COMBOBOX 1414, 88, 75, 89, 112, CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL | NOT WS_TABSTOP
     PUSHBUTTON "&Reguli de apel", 1108, 181, 75, 72, 14
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 8, 96, 245, 1
-    DEFPUSHBUTTON "&Apelează", 1590, 8, 103, 61, 14
-    PUSHBUTTON "A&nulează", 1591, 70, 103, 60, 14
+    DEFPUSHBUTTON "&Apelare", 1590, 8, 103, 61, 14
+    PUSHBUTTON "&Revocare", 1591, 70, 103, 60, 14
```

Comment:
This change introduces an accelerator collision with the line
 PUSHBUTTON "&Reguli de apel", 1108, 181, 75, 72, 14
Please check your romanian Windows to find out which of both lines need to be changed.